### PR TITLE
Add stories pages

### PR DESCRIPTION
**Pull request by fpapon**
This a WIP for adding a stories page to describe some project that is using Apache Karaf.

### CODE REVIEW

File: src/main/webapp/stories.html [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+            <div class="col">
+              <h4 class="pb-3 mb-4">Liferay</h4>
+              <p>Build portals, intranets, websites and connected experiences on the most flexible platform around.</p>
+              <a href="https://www.liferay.com/" target="_blank">https://www.liferay.com/</a>
+            </div>
+        </div>
+
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h4 class="pb-3 mb-4">Open Studio for ESB</h4>
```
Comment (jbonofre) [inline src/main/webapp/stories.html]: I would use `Talend ESB & Service platform` here.

File: src/main/webapp/stories.html [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+            <div class="col">
+              <h4 class="pb-3 mb-4">Fuse</h4>
+              <p>Enterprise Service Bus from RedHat.</p>
+              <a href="https://www.redhat.com/en/technologies/jboss-middleware/fuse/" target="_blank">https://www.redhat.com/en/technologies/jboss-middleware/fuse/</a>
+            </div>
+        </div>
+
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h4 class="pb-3 mb-4">WebSphere</h4>
```
Comment (jbonofre) [inline src/main/webapp/stories.html]: Do we have an approval from IBM ?

File: src/main/webapp/stories.html [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+        
+      </div><!-- Run on Karaf -->
+
+      <div class="container">
+
+        <!-- Support Karaf -->
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h2 class="pb-3 mb-4 font-italic border-bottom"><i class="fas fa-puzzle-piece"></i> Support Apache Karaf</h2>
+              <p>Frameworks that support Apache Karaf</p>
```
Comment (jbonofre) [inline src/main/webapp/stories.html]: I'm not sure about this section as we will probably miss some frameworks. This section could be tricky to maintain especially for frameworks "outside" of Apache foundation. Thoughts ?

**Review by jbonofre — CHANGES_REQUESTED** [review]

File: src/main/webapp/stories.html (hunk 2 above)
Comment (fpapon) [reply src/main/webapp/stories.html]: I don't think so, you're right, I will remove it.

File: src/main/webapp/stories.html (hunk 3 above)
Comment (fpapon) [reply src/main/webapp/stories.html]: You're right, I will remove it.

**Comment by fpapon** [comment]
I made the requested changes :)

**Comment by jbonofre** [comment]
@fpapon I guess we can remove the `WIP` flag on this PR, right ? Or do you have still some work in progress ?

**Comment by fpapon** [comment]
Yes, I removed.

**Comment by jbonofre** [comment]
LGTM, however, I would add just a note like this at the end of the `stories` page:

```
If you want to add your own application or solution powered or based on Apache Karaf, please create a website pull request or send a message on user@karaf.apache.org.
```
It would be great to have a "call to stories" sentence here.

Thoughts ?

**Comment by fpapon** [comment]
Agree about the sentence, I will make the changes

**Comment by fpapon** [comment]
Done, can you check please ?

**Review by jbonofre — APPROVED** [review]
LGTM, thanks !

**Pull request merged**